src/Netcarver/Textile/Parser.php

```diff
@@ -1981,7 +1993,7 @@ public function parse($text)
         $text = $this->retrieveTags($text);
         $text = $this->retrieveURLs($text);
 
-        $text = str_replace("<br />", "<br />\n", $text);
+        $text = (string) preg_replace("~<br[ ]*/?>~i", $this->getLineBreak()."\n", $text);
```

Comment:
This change is affecting existing embedded HTML snippets and dividers, by also changing their markup. To avoid this, this line should probably be:

```suggestion
        $text = str_replace($this->getLineBreak(), $this->getLineBreak()."\n", $text);
```

Which would leave user-input markup as-is. PHP-Textile does not change other void tags either, so for consistency you may not want to change br either.

Reply:
Unfortunately, I don't have a Docker setup for running the unit tests locally. Apologies for imposing on you with each successive small correction. Maybe we are nearly there 😬.